poetry.lock

```diff
@@ -1643,7 +1643,7 @@ files = [
 cffi = ">=1.4.1"
 
 [package.extras]
-docs = ["sphinx (>=1.6.5)", "sphinx_rtd_theme"]
+docs = ["sphinx (>=1.6.5)", "sphinx-rtd-theme"]
```

Comment:
Any idea why poetry seems to fight with itself about how to form this?

Reply:
It smells a bit like (#14079). Would guess this is related to a change [in Poetry 1.2.2](https://github.com/python-poetry/poetry/releases/tag/1.2.2). 

> Fix an issue where the names of extras were not normalized according to PEP 685 (https://github.com/python-poetry/poetry/pull/6541).

https://github.com/python-poetry/poetry/pull/6541 and https://github.com/python-poetry/poetry-core/pull/476

I'm surprised that dependabot has done this, but me running `poetry lock --no-update` locally in #14860 didn't.